src/legacy/ui/ui_render/bootstrap/template.js.hbs

```diff
@@ -74,6 +74,24 @@ if (window.__kbnStrictCsp__ && window.__kbnCspNotEnforced__) {
       });
     }
 
+    if ('{{appId}}' === 'core') {
+      return load({
+        deps: [
+          {
+            urls: [
+              '{{regularBundlePath}}/kbn-ui-shared-deps/{{sharedDepsFilename}}',
+            ]
+          }
+        ],
+        urls: [
+          '{{regularBundlePath}}/core/core.plugin.js',
```

Comment:
Really not familiar with this file, but is that really enough? The `dllJsChunks`, `'{{regularBundlePath}}/commons.bundle.js'` and `'{{dllBundlePath}}/vendors_runtime.bundle.dll.js'`  are all not required?

Reply:
Yep that's all that is needed. The DLLs are not required because `kbn/optimizer` builds each bundle as a separate build with all of it's dependencies self-contained (there is no longer a shared common chunk other than kbn-ui-shared-deps)

Reply:
> there is no longer a shared common chunk other than kbn-ui-shared-deps

(outside of this PR) I can understand the logic behind that, and it make sense for 3rd party plugins, but that mean that basically every 'kibana' client-side plugins bundles are shipped with `react` + `react-router-dom` + `...` contained? Correct me if I'm wrong, but atm a plugin can't have a different version of a given library, can it? We are increasing our total bundles size drastically with that decision, aren't we?

Reply:
Right now we have hand picked some dependencies to be shared via the `kbn-ui-shared-deps` module which webpack is configured to use for the hand-picked packages.